lib/bike_brigade_web/live/program_live/form_component.html.heex

```diff
@@ -147,6 +152,11 @@
             <:td let={item}>
               <%= item.category %>
             </:td>
+            <:td let={item} class="flex justify-center">
+              <%= if item.hidden do %>
+                <Heroicons.Outline.eye_off class="flex-shrink-0 w-6 h-6 mx-1 text-gray-600" />
+              <% end %>
+            </:td>
```

Comment:
```suggestion
            <:td let={item}>
              <%= if item.hidden do %>
                <div class="flex justify-center">
                  <Heroicons.Outline.eye_off class="flex-shrink-0 w-6 h-6 mx-1 text-gray-600" />
                </div>
              <% end %>
            </:td>
```

Reply:
I steered you wrong with adding `flex` to the td cell, instead we should have made a flex / justify-center container around the icon.